File: autoskola-central-back/src/main/java/hr/autoskola/model/Sat.java
package hr.autoskola.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Sat {

	private Integer satVoznjeId;
	private Integer polaznikId;
	private Integer voziloId;
	private Integer brojSata;
	private String opis; 
	private String datum;
	
	private Polaznik polaznik;
	private Vozilo vozilo;
}

File: autoskola-central-front/src/app/core/models/sat.ts
import { Vozilo } from "./vozilo";

export class Sat{
    satVoznjeId !: number;
    polaznikId !: number;
    voziloId !: number;
    brojSata !: number;
    opis !: string;
    datum !: string;

    vozilo !: Vozilo;
}